frontend/packages/operator-lifecycle-manager/integration-tests-cypress/tests/operator-install-global.spec.ts

```diff
@@ -104,15 +130,11 @@ xdescribe(`Interacting with a global install mode Operator (${operatorName})`, (
     );
   });
 
-  it(`uninstalls the operator`, () => {
-    cy.log('navigate to the operator uninstall modal in OperatorHub');
+  it(`uninstalls the Operator`, () => {
```

Comment:
Hi @rhamilto, we are running into the Cypress anti-pattern of [Having-tests-rely-on-the-state-of-previous-tests](https://docs.cypress.io/guides/references/best-practices.html#Having-tests-rely-on-the-state-of-previous-tests).  Here you have an `it` to create the operator, another `it` to test it's details, and a final `it` to uninstall the operator.  While this seems very logical, it is frowned upon by the Cypress gods :-)  All `its` should be able to run independently; you should be able to specify a `its.only` for any test and it should be able to run.  
To resolve I think these three tests should be in a single `it` test with alot of `cy.log` statements.  
From Cypress best practices about this:

"This ... is ideal because now we are resetting the state between each test and ensuring nothing in previous tests leaks into subsequent ones.
We’re also paving the way to make it less complicated to write multiple tests against the “default” state of the form. That way each test stays lean but each can be run independently and pass."

Reply:
I'm not opposed to this, but I was trying to maintain focus on getting the tests re-enabled.  This would need to be one `it` in order to comply with the best practice since the first `it` installs the operator.